### PR TITLE
fix editor colors

### PR DESCRIPTION
**Pull request by shivamhwp**
@AntoineKM 
Changed the colors a little bit so that they look good in both dark and white backgrounds.

### CODE REVIEW

**Comment by AntoineKM** [comment]
Hi @shivamhwp,

Thanks for your pull request! We will review it today!

In the meantime if you can give us a star it would be cool ahah

Have a nice day, we'll be back to you soon!

**Review by AntoineKM — CHANGES_REQUESTED** [review]
The colors are a bit bland, in general we are used to use flatter colors (https://flatuicolors.com/), could you either use flat colors, or use `theme.colors.accent.colorname`, or do the second solution which seems to me more time consuming but more judicious:
> Make a ``code-theme-dark.ts`` for the dark mode and a ``code-theme-light.ts`` for the light mode

I will be glad to help you if needed !

**Comment by shivamhwp** [comment]
where can I find the onThemeChange code, I'm trying to make two files for the themes but am unable to find the place where the theme code is written. Please help me with this.

File: docs/components/Playground/code-theme.ts [hunk 1]
```diff
@@ -78,13 +78,13 @@ const makeCodeTheme = (theme: DefaultTheme): PrismTheme => ({
...
       style: {
-        color: "#9EFFFF",
+        color: "#2980b9",
       },
     },
     {
       types: ["string", "attr-value"],
       style: {
-        color: "#A5FF90",
+        color: "#2ecc71", //#8D46AA
```
Comment (jerembdn) [inline docs/components/Playground/code-theme.ts:87]: remove useless comments

**Review by jerembdn — CHANGES_REQUESTED** [review]

File: docs/components/Playground/code-theme.ts (hunk 1 above)
Comment (shivamhwp) [reply docs/components/Playground/code-theme.ts]: ok

**Review by jerembdn — APPROVED** [review]

**Review by YounesBessa — APPROVED** [review]

File: docs/components/Playground/code-theme.ts [hunk 2]
```diff
@@ -84,8 +84,7 @@ const makeCodeTheme = (theme: DefaultTheme): PrismTheme => ({
     {
       types: ["string", "attr-value"],
       style: {
-        color: "#2ecc71", //#8D46AA
-      },
```
Comment (jerembdn) [inline docs/components/Playground/code-theme.ts:88]: It seems that you've deleted the closing brace here ?

**Review by jerembdn — CHANGES_REQUESTED** [review]

**Comment by shivamhwp** [comment]
@AntoineKM @jerembdn @YounesBessa 
Please merge the request.

**Review by jerembdn — APPROVED** [review]

**Comment by jerembdn** [comment]
> @AntoineKM @jerembdn @YounesBessa Please merge the request.

we're just waiting for @AntoineKM, it would be merged in some hours.

**Comment by shivamhwp** [comment]
> > @AntoineKM @jerembdn @YounesBessa Please merge the request.
> 
> we're just waiting for @AntoineKM, it will be merged in a few hours.

thank you very much.
it's my first open-source contribution.

**Comment by jerembdn** [comment]
> > > @AntoineKM @jerembdn @YounesBessa Please merge the request.
> > 
> > 
> > we're just waiting for @AntoineKM, it will be merged in a few hours.
> 
> thank you very much. it's my first open-source contribution.

you did good ! we really appreciate !
if you want to follow closer the advancement of the project, you can join your discord server, all links are on the [readme](/tonightpass/kitchen/#readme).
thanks :D

**Pull request merged**